### PR TITLE
Ratest 149: Check Modules Test 

### PR DESCRIPTION
**Pull request by insookwa**
I have made the required changes. kindly review @kdaud  @sherrif10 

https://issues.openmrs.org/browse/RATEST-149

### CODE REVIEW

File: ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-
 public class CheckModulesTest extends ReferenceApplicationTestBase {
 
     /**
      * Check the list of modules to be sure they are all started.
      */
-    @Ignore
-    @Category(org.openmrs.reference.groups.BuildTests.class)
+	@Test
+	@Category(BuildTests.class)
```
Comment (sherrif10) [inline ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java:20]: Thanks @insookwa , cc @dkayiwa  @ibacher  could you help us approve the man on this project such that we can see his builds

**Comment by kdaud** [comment]
@insookwa are sure that just removing the @ignore annotation passes the test?

File: ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java (hunk 1 above)
Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java:20]: > cc @dkayiwa @ibacher could you help us approve the man on this project such that we can see his builds

@sherrif10 before we call on these `devs` for approval, I would suggest we ensure that the `PR` author has simulated the test locally and its passing.

**Comment by kdaud** [comment]
@insookwa are sure that just removing the @ignore annotation passes the test?

File: ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java (hunk 1 above)
Comment (insookwa) [reply ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java:20]: @kdaud  I have tested on both chrome and firefox, both headless false and true

**Comment by insookwa** [comment]
> @insookwa are you sure that just removing the @ignore annotation passes the test?

yes @kdaud  it passes well locally in all conditions

**Comment by kdaud** [comment]
Waiting approval to run actions. cc: @dkayiwa @ibacher

**Comment by dkayiwa** [comment]
Approved the run actions.

**Comment by kdaud** [comment]
@insookwa its worth looking into the build failure at https://github.com/openmrs/openmrs-distro-referenceapplication/pull/518/checks?check_run_id=3212578526 and fix!

**Comment by kdaud** [comment]
Waiting approval to run actions!
cc: @dkayiwa

**Review by sherrif10 — COMMENTED** [review]
Add  breakpoint on line 21 , step over and then clearly check whether you can find and element called "moduleListing" from moduleListing, then if the debugger reaches on line 26. step in to see the values on that line, its where if its failing.Let me know  that helps

**Comment by sherrif10** [comment]
**for (WebElement eachModule : firstColumn) {
            // The name attr on the <input> elements should all be "stop" which indicates the module is correctly started.
            // If not, then grab the text from the 3rd column to show which module is not started.
            Assert.assertEquals("module not ready: " + eachModule.findElement(By.xpath("../../td[3]")).getText(), "stop", eachModule.getAttribute("name"));
        }
    }**

**Comment by kdaud** [comment]
Closing this PR since the test being addressed is already running in [master](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java)

**Pull request closed without merging**